<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorCalc GUI</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-graphics"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="paintTab">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <attribute name="title">
        <string>Расчет краски</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1,0">
        <item>
         <layout class="QHBoxLayout" name="pcComboBoxLayout">
          <item>
           <widget class="NamedField" name="paintPresetField" native="true"/>
          </item>
          <item>
           <widget class="NamedField" name="paintTypeField" native="true"/>
          </item>
          <item>
           <widget class="NamedField" name="materialTypeField" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
           <item>
            <layout class="QHBoxLayout" name="pcButtonLayout">
             <item>
              <widget class="QPushButton" name="pcCreatePresetButton">
               <property name="text">
                <string>Create</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pcUpdatePresetButton">
               <property name="text">
                <string>Update</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pcRemovePresetButton">
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
          <property name="horizontalSpacing">
           <number>64</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="1" column="0">
           <widget class="NamedField" name="pcDividerField" native="true"/>
          </item>
          <item row="2" column="0">
           <widget class="NamedField" name="pcPercentageField" native="true"/>
          </item>
          <item row="0" column="0">
           <widget class="NamedField" name="pcConsumptionField" native="true"/>
          </item>
          <item row="2" column="1">
           <widget class="NamedField" name="pcCirculationField" native="true"/>
          </item>
          <item row="1" column="1">
           <widget class="NamedField" name="pcSheetLengthField" native="true"/>
          </item>
          <item row="0" column="1">
           <widget class="NamedField" name="pcSheetWidthField" native="true"/>
          </item>
          <item row="3" column="1">
           <widget class="NamedField" name="pcPaintReserveField" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="horizontalWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3,2">
           <item>
            <widget class="QPushButton" name="pcCalculateButton">
             <property name="text">
              <string>Рассчитать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="NamedField" name="pcResultField" native="true"/>
           </item>
           <item>
            <widget class="QPushButton" name="pcClearFieldsButton">
             <property name="text">
              <string>Очистить поля</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="lacquerTab">
       <attribute name="title">
        <string>Расчет лака</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,1,0">
        <item>
         <layout class="QHBoxLayout" name="lcComboBoxLayout">
          <item>
           <widget class="NamedField" name="lacquerPresetField" native="true"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="lcButtonLayout">
            <item>
             <widget class="QPushButton" name="lcCreatePresetButton">
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="lcUpdatePresetButton">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="lcRemovePresetButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
          <property name="horizontalSpacing">
           <number>64</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="NamedField" name="lcConsumptionField" native="true"/>
          </item>
          <item row="0" column="1">
           <widget class="NamedField" name="lcSheetWidthField" native="true"/>
          </item>
          <item row="1" column="0">
           <widget class="NamedField" name="lcPercentageField" native="true"/>
          </item>
          <item row="2" column="0">
           <widget class="NamedField" name="lcCirculationField" native="true"/>
          </item>
          <item row="1" column="1">
           <widget class="NamedField" name="lcSheetLengthField" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="horizontalWidget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,3,2">
           <item>
            <widget class="QPushButton" name="lcCalculateButton">
             <property name="text">
              <string>Рассчитать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="NamedField" name="lcResultField" native="true"/>
           </item>
           <item>
            <widget class="QPushButton" name="lcClearFieldsButton">
             <property name="text">
              <string>Очистить поля</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="foilTab">
       <attribute name="title">
        <string>Расчет фольги</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,1,0">
        <item>
         <layout class="QHBoxLayout" name="fcComboBoxLayout">
          <item>
           <widget class="NamedField" name="foilPresetField" native="true"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="fcButtonLayout">
            <item>
             <widget class="QPushButton" name="fcCreatePresetButton">
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="fcUpdatePresetButton">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="fcRemovePresetButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0">
          <property name="horizontalSpacing">
           <number>64</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="NamedField" name="fcLengthField" native="true"/>
          </item>
          <item row="0" column="1">
           <widget class="NamedField" name="fcLengthReserveField" native="true"/>
          </item>
          <item row="1" column="0">
           <widget class="NamedField" name="fcWidthField" native="true"/>
          </item>
          <item row="2" column="0">
           <widget class="NamedField" name="fcSheetNumberField" native="true"/>
          </item>
          <item row="1" column="1">
           <widget class="NamedField" name="fcWidthReserveField" native="true"/>
          </item>
          <item row="2" column="1">
           <widget class="NamedField" name="fcCirculationField" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="horizontalWidget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,3,2">
           <item>
            <widget class="QPushButton" name="fcCalculateButton">
             <property name="text">
              <string>Рассчитать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="NamedField" name="fcResultField" native="true"/>
           </item>
           <item>
            <widget class="QPushButton" name="fcClearFieldsButton">
             <property name="text">
              <string>Очистить поля</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="foilRollsTab">
       <attribute name="title">
        <string>Рулоны фольги</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0">
        <item>
         <layout class="QHBoxLayout" name="frComboBoxLayout">
          <item>
           <widget class="NamedField" name="foilRollsPresetField" native="true"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="frButtonLayout">
            <item>
             <widget class="QPushButton" name="frCreatePresetButton">
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="frUpdatePresetButton">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="frRemovePresetButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,1">
            <item>
             <widget class="NamedField" name="frLengthField" native="true"/>
            </item>
            <item>
             <widget class="NamedField" name="frWidthField" native="true"/>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="NamedField" name="frSuitableRollsField" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="horizontalWidget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,2">
           <item>
            <widget class="QPushButton" name="frFindSuitableRollsButton">
             <property name="text">
              <string>Рассчитать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="frClearFieldsButton">
             <property name="text">
              <string>Очистить поля</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NamedField</class>
   <extends>QWidget</extends>
   <header>namedfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
